src/components/Slider/Slider.js

```diff
@@ -66,6 +66,11 @@ export default class Slider extends PureComponent {
 
     requestAnimationFrame(() => {
       this.setState((prevState, props) => {
+        if (evt && evt.target.classList.contains('bx-slider-text-input')) {
```

Comment:
Is it possible that `evt.target` could be undefined here?

Reply:
Another thing I'm wondering is if `evt.target` could be a SVG: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/1173756/

Reply:
Would it be good to just change this to :

```js
if (evt && evt.target && evt.target.classList.contains('bx-slider-text-input'))
```

Reply:
Sounds good to me 😄

Reply:
👍